defaults/main.yml

```diff
@@ -103,6 +103,7 @@ gitlab_project_group: test_group
 gitlab_project_group_id: 1
 
 # gitlab webhooks
+gitlab_allow_local_requests: False
```

Comment:
Please set to True in molecule playbooks

Reply:
resolve in e2e411f